spec/index.html

```diff
@@ -5465,9 +5465,10 @@ <h5>RDFterm-equal</h5>
             <p>This function is defined as follows:</p>
             <ul>
                 <li>Returns TRUE if <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms, as defined below.</li>
-                <li>Produces a type error if <code>term1</code> and <code>term2</code> are both literals that have the
+                <li>Produces a type error if <code>term1</code> and <code>term2</code> are both literals such that these two literals have the
               same <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>, this datatype IRI is <em>not</em> in the
-              set of <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>.</li>
+              set of <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>, and the
+              <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical forms</a> of the two literals are different from one another.</li>
```

Comment:
```suggestion
                <li>Produces a type error if <code>term1</code> and <code>term2</code> are both literals having the
              same <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>; this datatype IRI is <em>not</em> in the
              set of <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>; and the
              <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical forms</a> of the two literals are different from one another.</li>
```

Reply:
@hartig Since you suggested this text before, do you agree with this change? (feel free to commit directly if so)

Reply:
@rubensworks @hartig 

GitHub seems to be confused about this PR. The conversation [seen here, in the _Conversation_ tab](https://github.com/w3c/sparql-query/pull/17) does not track with the conversation [seen on the _Files changed_ tab](https://github.com/w3c/sparql-query/pull/17/files).

I have resuggested the same change, against the current state of the file. Maybe [this](https://github.com/w3c/sparql-query/pull/17#pullrequestreview-1339564323) can be merged (and [this](https://github.com/w3c/sparql-query/pull/17#discussion_r1132788329) can be marked resolved)?

Reply:
I am marking [this change suggestion](https://github.com/w3c/sparql-query/pull/17#discussion_r1132788329) as resolved by merging [the other one](https://github.com/w3c/sparql-query/pull/17#pullrequestreview-1339564323).